src/nvim/CMakeLists.txt

```diff
@@ -85,6 +85,10 @@ file(GLOB NVIM_HEADERS *.h)
 file(GLOB XDIFF_SOURCES xdiff/*.c)
 file(GLOB XDIFF_HEADERS xdiff/*.h)
 
+file(GLOB TREESITTER_SOURCES ../tree_sitter/*.c)
+file(GLOB TS_SOURCE_AMALGAM ../tree_sitter/lib.c)
+list(REMOVE_ITEM TREESITTER_SOURCES ${TS_SOURCE_AMALGAM})
```

Comment:
should we instead delete the amalgamation from the tree? or is it used somewhere?

Reply:
it would make the steps for upgrading tree-sitter sources slightly more complicated, though perhaps it doesn't matter as it is pretty fail-fast if you include the file in an update.